bootq/src/main/java/life/genny/bootq/models/BatchLoading.java

```diff
@@ -227,6 +234,12 @@ public void persistDefBaseEntityAttributes(Map<String, Map<String, String>> proj
 		DataType dttBoolean = attributeUtils.getDataType(realmName, "DTT_BOOLEAN", false);
 		DataType dttText = attributeUtils.getDataType(realmName, "DTT_TEXT", false);
 
+        // above methods have a chance of returning null
+        if(dttBoolean == null)
```

Comment:
Fail fast - the dttBoolean check & exception throwing could be moved before dttText is retrieved

Reply:
Exception thrown now in getDataType